src/styles/SelectedRevsStyle.ts

```diff
@@ -94,10 +92,6 @@ export const SelectRevsStyles = (mode: string) => {
                 : Colors.SecondaryTextDark,
             },
 
-            svg: {
-              marginRight: `${Spacing.xSmall}px`,
-              fontSize: '1rem',
```

Comment:
I think this is still needed, otherwise the icons are too small and too close to the text. There might be other ways to do it later (like providing them on the SVG icon themselves) but let's keep it for now.

See:
Before:
![image](https://github.com/user-attachments/assets/a9e9df01-3193-4493-82e4-363985726916)

After:
![image](https://github.com/user-attachments/assets/e5a9dca6-2fdf-4d6d-a0b3-e86eeb479feb)

Reply:
Okay I missed that :) I thought it was unused. Adding it back now